Mathlib/Data/Nat/Cast/NeZero.lean

```diff
@@ -3,15 +3,20 @@ Copyright (c) 2014 Mario Carneiro. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro, Gabriel Ebner
 -/
+import Mathlib.Data.Nat.Defs
 import Mathlib.Data.Nat.Cast.Defs
 import Mathlib.Algebra.NeZero
 
 /-!
 # Lemmas about nonzero elements of an `AddMonoidWithOne`
 -/
 
+open Nat
+
 namespace NeZero
 
+theorem _root_.NeZero.one_le {n : ℕ} [NeZero n] : 1 ≤ n := one_le_iff_ne_zero.mpr (NeZero.ne n)
```

Comment:
```suggestion
theorem _root_.NeZero.one_le {n : ℕ} [NeZero n] : 1 ≤ n := by have := NeZero.ne n; omega
```
allows you to avoid introducing the import here.